### PR TITLE
feat(quic): more listener options

### PR DESCRIPTION
**Pull request by qzhuyan**

### CODE REVIEW

File: apps/emqx/i18n/emqx_schema_i18n.conf [hunk 1]
```diff
@@ -1980,6 +1979,36 @@ message within this interval.
...
+        en: """
+How long a handshake can idle before it is discarded.
+"""
+        zh: """
+一个握手在被丢弃之前可以空闲多长时间。
+"""
+    }
+    label: {
+        en: "Handshake Idle Timeout"
+        zh: "握手发呆超时时间"
```
Comment (lafirest) [inline apps/emqx/i18n/emqx_schema_i18n.conf:1993]: maybe 「空闲」will be better?

File: apps/emqx/i18n/emqx_schema_i18n.conf [hunk 2]
```diff
@@ -1980,6 +1979,36 @@ message within this interval.
...
+        en: """
+How often to send PING frames to keep a connection alive. 0 means disabled.
+"""
+        zh: """
+发送 PING 帧的频率，以保活连接. 设为0，禁用
+"""
+    }
+    label: {
+        en: "Keep Alive Interval"
+        zh: "PING保活频率"
```
Comment (lafirest) [inline apps/emqx/i18n/emqx_schema_i18n.conf:2008]: nit : PING 保活频率

File: apps/emqx/i18n/emqx_schema_i18n.conf (hunk 1 above)
Comment (terry-xiaoyu) [reply apps/emqx/i18n/emqx_schema_i18n.conf:1993]: 发呆可以的吧

File: apps/emqx/i18n/emqx_schema_i18n.conf (hunk 2 above)
Comment (terry-xiaoyu) [inline apps/emqx/i18n/emqx_schema_i18n.conf:2008]:
```suggestion
        zh: "PING 保活频率"
```

File: apps/emqx/i18n/emqx_schema_i18n.conf (hunk 1 above)
Comment (qzhuyan) [reply apps/emqx/i18n/emqx_schema_i18n.conf:1993]: I just copied from others.

**Pull request merged**